packages/grafana-ui/src/components/Alert/Alert.story.tsx

```diff
@@ -21,172 +20,61 @@ const meta: ComponentMeta<typeof Alert> = {
     knobs: {
       disable: true,
     },
-    controls: {
-      exclude: ['onRemove'],
-    },
+    controls: {},
   },
   argTypes: {
     severity: { control: { type: 'select', options: severities } },
+    onRemove: {
+      description: "To show a button or a close icon, select 'function'. To get rid of them, select 'undefined'",
+      control: { type: 'select', options: [action('Remove button clicked'), undefined] },
+    },
+    buttonContent: {
+      description: "To show an example of button, select 'Close', otherwise, select 'undefined'",
+      control: { type: 'select', options: ['Close', undefined] },
+    },
   },
 };
 
-export const Examples: ComponentStory<typeof Alert> = ({ severity, title, buttonContent }) => {
+export const InlineBanner: ComponentStory<typeof Alert> = ({ severity, title, buttonContent, onRemove }) => {
+  return (
+    <Alert
+      title={title}
+      severity={severity}
+      buttonContent={buttonContent ? <span>{buttonContent}</span> : ''}
+      onRemove={onRemove ? onRemove : undefined}
+    >
+      <VerticalGroup>
+        <div>Child content that includes some alert details, like maybe what actually happened.</div>
+      </VerticalGroup>
+    </Alert>
+  );
+};
+
+InlineBanner.args = {
+  severity: 'error',
+  title: 'Title',
+  buttonContent: 'Close',
+  onRemove: action('Remove button clicked'),
+};
+
+InlineBanner.parameters = {
+  controls: { expanded: true },
+};
+
+export const Toast: ComponentStory<typeof Alert> = ({ severity }) => {
```

Comment:
same as above... think this should just be `args` and they should just be spread onto the `Alert` component below